Public/Src/FrontEnd/TypeScript.Net/TypeScript.Net.UnitTests/Parsing/Expressions.cs

```diff
@@ -50,6 +50,12 @@ public void ConditionalExpression()
             Assert.Equal(code, node.GetFormattedText());
         }
 
+        [Fact]
+        public void SwitchExpression()

```

Comment:
>SwitchExpression() [](start = 20, length = 18)

TODO?

Reply:
OOps, created seperate file for the tests, was initially planning to put here..

---
In reply to: [274109954](https://github.com/Microsoft/BuildXL/pull/119#discussion_r274109954) [](ancestors = 274109954)